test/tests/autotranslation.js

```diff
@@ -18,16 +18,10 @@ var BeakerPageObject = require('./beaker.po.js');
 var path = require('path');
 
 describe('autotranslation', function() {
-  beforeEach(function(done) {
+  beforeEach(function() {
     jasmine.DEFAULT_TIMEOUT_INTERVAL = 120000; // Slow initialization cells, CI box might need a long time to initialize these
     beakerPO = new BeakerPageObject();
-    browser.get(beakerPO.baseURL).then(
-        function(){done();},
-        function(reason){ console.log('error occurred while browser load page : ' + reason);}
-    );
-    browser.waitForAngular().then(function() {
-      done();
-    });
+    browser.get(beakerPO.baseURL);
```

Comment:
I still think we should try doing it async, without it we're pretty much back where we started. So if you put the done callback in and add this, we should be good:

```javascript
browser.get(beakerPO.baseURL).then(done);
```

Also, maybe there's a way to write that beforeEach only once? Tests are running in a node environment so it should be easy to isolate it in a module and import.